test/e2e/manifests/components/base/kustomization.yaml

```diff
@@ -10,8 +12,3 @@ patches:
 - path: workflow-controller-deployment.yaml
 - path: workflow-controller-cluster-workflow-template-rbac.yaml
 - path: minio-deployment.yaml
-
-labels:
-- includeSelectors: true
-  pairs:
-    app.kubernetes.io/part-of: argo
```

Comment:
FYI: This label was only needed when using the old `--prune` method. When using [apply sets](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/declarative-config/#alternative-kubectl-apply-f-directory-prune), Kubernetes automatically generates the appropriate label.